packages/hello-world-react/app/ssr.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+
+/* global WEBPACK_PACKAGE_JSON_MOBIFY */
+
+import path from "path";
+import {
+  createApp,
+  createHandler
+} from "pwa-kit-react-sdk/ssr/server/express";
+import { render } from "pwa-kit-react-sdk/ssr/server/react-rendering";
```

Comment:
For some reason we're using a different formatting for this file, note the `;` at the end of each line. 

Is that intended?